### ISSUE TITLE
A11y_VSCode_Untitled tab_HighContrast: In high contrast mode, focus is not visible on "untitled" tab

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#27271]
“[Check out Accessibility Insights](https://accessibilityinsights.io/)! – Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:** -A11yMAS; -A11yTCS; -A11ySev2;-BM-VisualStudioCode-Win32-Nov2021;-Visual Studio Code Client;-Win11;-FTP;-DesktopApp;

-- Environment:
Visual Studio Code – Insider
Version: 1.79.0-insider (user Setup)
OS Version: Microsoft Windows 11 Enterprise (22621.1702)

-- Pre Requisite:

Change system high contrast to aquatic.

-- Repro Steps:

1. Open Visual Studio Code in your system
2. Tab to the explorer and press ctrl+ alt+windows+n to add a new text file
3. press ctrl+M to activate the tab navigation
4. use shift tab for the editor of the text file and move the close button of the untitled tab.
5. Use one more shift tab and check if the keyboard focus is visible on the screen.

-- Actual Result:
In high contrast mode, focus is not visible on "untitled-1" tab

-- Expected Result:
In high contrast mode, focus should be visible on "untitled-1" tab

-- User Impact:
Keyboard users and low vision users will finds difficult while working if they are unable to see the focus visibility on "untitled-1" tab

-- Attachments:
